main.go

```diff
@@ -101,6 +106,21 @@ func StartCmd() cli.Command {
 				EnvVar: EnvServiceAccountName,
 				Value:  DefaultServiceAccount,
 			},
+			cli.StringFlag{
+				Name:  FlagHelperPodFile,
+				Usage: "Paths to the Helper pod yaml file",
+				Value: "",
+			},
+			cli.StringFlag{
+				Name:  FlagNodePathType,
+				Usage: "Paths to the Helper pod yaml file",
+				Value: string(v1.HostPathDirectoryOrCreate),
+			},
+			cli.StringFlag{
+				Name:  FlagNodePathMountPropagation,
+				Usage: "Paths to the Helper pod yaml file",
```

Comment:
Wrong usage. And this should be another PR.

Also, I am not sure why we need mount propagation for the helper pod. @mgoltzsche can you explain your use case?

Reply:
@yasker my use case is [this](https://github.com/mgoltzsche/cache-provisioner): provision a cache as container image by running buildah within the helper pod to provision (and commit on PV deletion) the PV directory using overlayfs. However the overlayfs mount must be propagated from the helper pod to the host so that its contents can be mounted as volume to another pod after the helper pod terminated.

Reply:
@yasker @mgoltzsche  i removed mount propagation and node path type flag in this pr, after this pr is done, i will add these flags in next pr.

Reply:
@nicktming I'd do the next PR since other requirements came up and I could provide a corresponding usage example as well.